exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function ShouteyGreeting(name){
+    return (console.log("HELLO "+ name.toUpperCase()));
```

Comment:
I like the way how short is your function and has only one line of code, brilliant .